de.dlr.sc.virsat.cef.swtbot.test/src/de/dlr/sc/virsat/cef/swtbot/test/ProjectUiAllTest.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.cef.swtbot.test;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+/**
+ * 
+ * @author muel_s8
```

Comment:
Remove author, we try to avoid it.

Reply:
- Since CEF is derived from core, all core functionality in CEF should have already been thoroughly covered by test cases in core already. I think we should only test for CEF specific functionality here.

- Currently this is a SWTBot stub without Graphiti. No further adjustments were needed. As soon as a Graphiti stub is added, this action will be necessary. As for now I wouldn't like to introduce not yet used dependencies.